src/universal/components/UserMenu/UserMenu.js

```diff
@@ -11,18 +13,16 @@ export default class UserMenu extends Component {
   }
   render() {
     const props = this.props;
-    const menuItem = (id, label) => {
-      return (
-        <a className={styles.item} href="#" key={id} title={label}>{label}</a>
-      );
-    };
-    const mapMenuItems = (menuItems) => {
-      return (
-        menuItems.map(item =>
-          menuItem(item.id, item.label)
-        )
-      );
-    };
+    // TODO: Not sure why eslint doesn't like this block
+    const menuItem = (id, label) => (
```

Comment:
again, we'll wanna move these into methods. a function declared in a render is going to be recreated each time. that means anything that depends on that will be new, thus triggering a rerender.